### PR TITLE
Changes for the new platform.

### PR DESCRIPTION
**Pull request by hascar**
Changes for the new platform because master is still used for old platform.

The tests are running here:
  * https://ci.openquake.org/job/zdevel_oq-platform2/1024/
  * https://ci.openquake.org/job/zdevel_oq-platform-standalone/314/

### CODE REVIEW

File: openquakeplatform_taxtweb/templates/taxtweb/index.html [hunk 1]
```diff
@@ -24,6 +24,9 @@
     {{block.super}}
 
 {% include "taxtweb/incl_taxtweb_inlinecss.html" %}
+ <div id="oq-body-heading">
+        <!--<h1>TaxTweb - Graphical tool for editing GEM Taxonomy strings</h1> -->
```
Comment (nastasi-oq) [inline openquakeplatform_taxtweb/templates/taxtweb/index.html]: Commented title ?

Comment (hascar) [reply openquakeplatform_taxtweb/templates/taxtweb/index.html]: This must be deleted

Comment (hascar) [reply openquakeplatform_taxtweb/templates/taxtweb/index.html]: Solved with this https://github.com/gem/oq-platform-taxtweb/pull/27/commits/f1a50f79420797526eed5f9c4e4bf8b88de0b3bf

**Comment by nastasi-oq** [comment]
Please, factorize the added piece of code.

**Review by nastasi-oq — APPROVED** [review]
LGTM

**Pull request merged**